backend/benefit/requirements.in

```diff
@@ -5,15 +5,15 @@ django-cors-headers
 django-environ
 django-extensions
 django-filter
-django-localflavor~=3.1
+django-localflavor
```

Comment:
Have you checked the django-localflavor use in the benefit project? I think there might be a reason why it had been set to ~=3.1. I think there's some IBAN related custom class deriving from django-localflavor if I remember correctly... yes, LocalizedIBANValidator and LocalizedIBANField. Do these still work after the django-localflavor upgrade?

Reply:
3.1 is the most recent version, so it's working now. I think that if future updates broke the derived classes, then the derived classes should be fixed.